src/foam/nanos/approval/ApprovalRequest.js

```diff
@@ -659,14 +667,40 @@
             // If the dif of objects is calculated and stored in Map(obj.propertiesToUpdate),
             // this is for updating object approvals
             if ( obj.propertiesToUpdate ) {
-              // then here we created custom view to display these properties
-              X.stack.push({
-                class: 'foam.nanos.approval.PropertiesToUpdateView',
-                propObject: obj.propertiesToUpdate,
-                objId: obj.objId,
-                daoKey: obj.daoKey,
-                title: 'Updated Properties and Changes'
-              });
+              if ( obj.operation === foam.nanos.ruler.Operations.CREATE ){
+                var temporaryNewObject = obj.of.create({}, X);
+
+                var propsToUpdate = obj.propertiesToUpdate;
+
+                var keyArray = Object.keys(propsToUpdate);
+
+                for ( var i = 0; i < keyArray.length; i++ ){
+                  temporaryNewObject[keyArray[i]] = propsToUpdate[keyArray[i]];
+                }
```

Comment:
Yeah so new version we have to iterate over the properties check if transient and then remove it out anyways. So we can stick with the for loop because I'd have to do it anyways and iterate over the propsToUpdate to check if its transient to do copyFrom